### DISCUSSION TITLE
is the older version better?

### DISCUSSION THREAD

**Discussion by xiangbohua** [Sequel-Ace/Sequel-Ace#1010] [General] [answered]
I feel colored tab was batter than colored spot.

**Comment by Jason-Morcos** [comment, marked as answer]
See #1013

**Reply by xiangbohua** [reply]
Thank for reply.
The way to color a tab may not be a big deal but why remove the shortcut for filter by tap Command + F?
Or did i miss some configuration?
Anyway Sequel is my favorite client.

**Reply by Kaspik** [reply]
That's also fixed - see https://github.com/Sequel-Ace/Sequel-Ace/issues/1014

**Reply by Kaspik** [reply]
Did you install 3.3.1 or search through the issues? I believe all this was fixed already.
